packages/data-extensions/src/surfaces/admin/api/standard/standard.ts

```diff
@@ -0,0 +1,33 @@
+import {ApiVersion, Auth, GraphQLError, I18n} from '../../../../api';
+import type {ExtensionTarget as AnyExtensionTarget} from '../../extension-targets';
+
+/**
+ * The following APIs are provided to all extension targets.
+ */
+export interface StandardApi<ExtensionTarget extends AnyExtensionTarget> {
```

Comment:
I think we're missing `data: {selected: [{id: string}]}`?

Reply:
I added it `Data` to the shared APIs and pulled it in from there. We should note that in UI extensions it's not part of the standard API. It's added to action, block, and print individually, but I think it's okay to deviate here. Thoughts?